tests/test_encryption.py

```diff
@@ -70,6 +86,8 @@ def test_encryption(name):
     ],
 )
 def test_both_password(name, user_passwd, owner_passwd):
+    if not HAS_PYCRYPTODOME:
+        return
```

Comment:
I like using [`pytest.skip`](https://docs.pytest.org/en/7.1.x/reference/reference.html#pytest-skip) vs a `return` to make the intention clearer. Ideally, could use `pytest.mark.skipif` annotation, but not sure can use it and `parametrize` at the same time.

Reply:
Very good point :+1: 

pytest.mark.skipif and parametrize can for sure be used at the same time... but I don't know how to use variables from paremtrize within the skipif condition. I think that is not possible

Reply:
All of these cases are using a global module constant, and not anything from the `parameterize` clause, so should be good then to combine?